Build failure when using src/set-project-version.sh: Project version is inherited from parent.

**Describe the bug**
When using `src/set-project-version.sh` on the master branch, goal `org.codehaus.mojo:versions-maven-plugin:2.8.1:set` will throw a build failure: Project version is inherited from parent.

**To Reproduce**
Steps to reproduce the behavior:
1. Check out the master branch
2. run `src/set-project-version.sh 2.8.0-rc-202106042216`

**Expected behavior**
No failure happened

**Screenshots**
```
[INFO] ------------< org.apache.pulsar:pulsar-presto-distribution >------------
[INFO] Building Pulsar SQL :: Pulsar Presto Distribution 2.8.0-rc-202106042216
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- versions-maven-plugin:2.8.1:set (default-cli) @ pulsar-presto-distribution ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.078 s
[INFO] Finished at: 2021-06-05T10:15:04+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:versions-maven-plugin:2.8.1:set (default-cli) on project pulsar-presto-distribution: Project version is inherited from parent. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

--- a/src/set-project-version.sh
+++ b/src/set-project-version.sh
@@ -38,7 +38,6 @@ OLD_VERSION=`python ${ROOT_DIR}/src/get-project-version.py`
 
 mvn versions:set -DnewVersion=$NEW_VERSION
 mvn versions:set -DnewVersion=$NEW_VERSION -pl buildtools
-mvn versions:set -DnewVersion=$NEW_VERSION -pl pulsar-sql/presto-distribution
 # Set terraform ansible deployment pulsar version
 sed -i -e "s/${OLD_VERSION}/${NEW_VERSION}/g" ${TERRAFORM_DIR}/deploy-pulsar.yaml
 

